lib/dogapi/v1/user.rb

```diff
@@ -23,6 +26,87 @@ def invite(emails, options = {})
           suppress_error_if_silent e
         end
       end
+
+      # Create a user
+      #
+      # :description => Hash: user description containing 'handle' and 'name' properties
+      def create_user(description = {})
+        begin
+          params = {
+            :api_key => @api_key,
+            :application_key => @application_key
+          }
+
+          request(Net::HTTP::Post, "/api/#{API_VERSION}/user", params, description, true)
+        rescue Exception => e
+          suppress_error_if_silent e
+        end
+      end
+
+      # Retrieve user information
+      #
+      # :handle => String: user handle
+      def get_user(handle)
+        begin
+          params = {
+            :api_key => @api_key,
+            :application_key => @application_key
+          }
+
+          request(Net::HTTP::Get, "/api/#{API_VERSION}/user/#{handle}", params, nil, false)
+        rescue Exception => e
+          suppress_error_if_silent e
+        end
+      end
+
+      # Retrieve all users
+      def get_all_users()
+        begin
+          params = {
+            :api_key => @api_key,
+            :application_key => @application_key
+          }
+
+          request(Net::HTTP::Get, "/api/#{API_VERSION}/user", params, nil, false)
+        rescue Exception => e
+          suppress_error_if_silent e
+        end
+      end
+
+      # Update a user
+      #
+      # :handle => String: user handle
+      # :description => Hash: user description optionally containing 'name', 'email',
+      # 'is_admin', 'disabled' properties
+      def update_user(handle, description = {})
+        begin
+          params = {
+            :api_key => @api_key,
+            :application_key => @application_key
+          }
+
+          request(Net::HTTP::Put, "/api/#{API_VERSION}/user/#{handle}", params, description, true)
+        rescue Exception => e
+          suppress_error_if_silent e
+        end
+      end
+
+      # Disable a user
+      #
+      # :handle => String: user handle
+      def disable_user(handle)
```

Comment:
Wondering if `disable_user` is maybe a misleading name given that this _is_ the delete operation. Once disabled the user would be gone right? Maybe delete_user would be a better name?